apluslms_roman/backends/__init__.py

```diff
@@ -1,19 +1,21 @@
+import logging
 from collections import namedtuple
 from collections.abc import Mapping
 
 from ..observer import BuildObserver
-
+from ..utils.path_mapping import get_host_path
 
 BACKENDS = {
     'docker': 'apluslms_roman.backends.docker.DockerBackend',
 }
 
-
 BuildTask = namedtuple('BuildTask', [
     'path',
     'steps',
 ])
 
+logger = logging.getLogger(__name__)
```

Comment:
This should be the first thing after imports. Thus, imports > logger > BACKENDS > BuildTask ..